packages/runtime/plugin-testing/src/cli/index.ts

```diff
@@ -128,6 +128,9 @@ export const testingPlugin = (): CliPlugin<{
               `<rootDir>/src/**/*.test.[jt]s?(x)`,
               `<rootDir>/tests/**/*.test.[jt]s?(x)`,
             ],
+            transformIgnorePatterns: [
```

Comment:
Some users may not use pnpm, so we should also match paths of npm and yarn.

Reply:
I'll try npm and yarn.

Reply:
Done. You can check it.